lib/iron_bank/metadata.rb

```diff
@@ -12,6 +12,13 @@ def excluded_fields
       fields.fetch(object_name, [])
     end
 
+    # NOTE: For some resources, fields are queryable with some restrictions,
+    #       e.g. the `Invoice#body` can only be added to the list of fields if
+    #       there is only one invoice in the query response.
+    def separate_query_fields
```

Comment:
WDYTA killing this "abstract" method, and letting the snowflakes deal with it?

Reply:
The `IronBank::Describe::ExcludedFields` service class needs to call that public class method for each resource. Hence, I believe we should keep it as a public method that all resources can respond to, and let the specific classes handle their snowflakes.